binarylog/grpc_binarylog_v1/binarylog.pb.go

```diff
@@ -430,7 +430,7 @@ type ClientHeader struct {
 	MethodName string `protobuf:"bytes,2,opt,name=method_name,json=methodName,proto3" json:"method_name,omitempty"`
 	// A single process may be used to run multiple virtual
 	// servers with different identities.
-	// The authority is the name of such a server identitiy.
+	// The authority is the name of such a server identity.
```

Comment:
We should not be modifying this `pb.go` as this is generated from protos in the grpc-proto repo. And this comment comes from [here](https://github.com/grpc/grpc-proto/blob/master/grpc/binlog/v1/binarylog.proto#L123).

Let us rollback the changes to this file and cut a PR to fix the typo in the proto definition in the gRPC-proto repo. This will fix it the next time we generate.